WDAC-Policy-Wizard/app/MSIX/AppxManifest.xml

```diff
@@ -4,7 +4,7 @@
   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
   <Identity Name="Microsoft.WDAC.WDACWizard"
-            Version="1.6.6.0"
```

Comment:
If you're using semantic versioning (which this is close to, but not exactly)

first number is for API/ABI changes (increment if you break that)
second number is for new features.
third number is for bugs fixed.